### ISSUE TITLE
Kopernicus not generating cache files.

### ISSUE THREAD

**Issue by electricpantsGITHUB** [Kopernicus/Kopernicus#163]
I have nothing but a mod I'm working on, and Kopernicus (ModularFlightIntegrator too, bcause that comes packaged with Kopernicus) isn't generating a cache file as ProtoJeb21 said it would. I've also checked, and the filepath to where the file is supposed to be generated is correct.

Can anyone figure this out?

.cfg of the planet:

@Kopernicus:AFTER[Kopernicus]
{
	Body
	{
		name = Abraxa
		flightGlobalsIndex = 805
		cacheFile = TDS/PhoenixSystem/Abraxa/Cache/Abraxa.bin
		
       	Template
        {
 			name = Laythe
			removeAllPQSMods = true
			removeOcean = true
        }
		Orbit
		{
			referenceBody = Phoenix
			inclination = 1.15
			eccentricity = 0.012
			semiMajorAxis = 1.44e+9
			longitudeOfAscendingNode = 309
			argumentOfPeriapsis = 23
			meanAnomalyAtEpoch = 114
			epoch = 92
			color = 1,1,1,1
		}
		Properties
		{
			description = Abraxa is a cold, dark world with a hot side covered in liquid ammonia. The planet is smack in the middle of the ammonia-based habitable zone of Phoenix, so it's feasible for hosting alien life, and Abraxa will likely be the last refuge for kerbalkind once the Sun dies.
			radius = 779000
			geeASL = 1.44
			rotationPeriod = 
			rotates = true
			tidallyLocked = true
			initialRotation = 0
			isHomeWorld = false
			timewarpAltitudeLimits = 0 1000 20000 45000 60000 70000 80000 90000

			biomeMap = TDS/PhoenixSystem/Abraxa/PluginData/Abraxa_biome.png

			Biomes
			
			Biome
			{
				name = Ammonia Ocean
				value = 0.5
				color = 1,0,0,1
			}
			Biome
			{
				name = Shore
				value = 1
				color = 0,1,0,1
			}
			Biome
			{
				name = Ice Sheet
				value = .5
				color = 0,0,1,1
			}
			Biome
			{
				name = Giant Crater
				value = .5
				color = 1,1,0,1
			}
		}
		ScienceValues
		{
			flyingLowDataValue = 8
			flyingHighDataValue = 7
			landedDataValue = 10
			inSpaceLowDataValue = 7
			inSpaceHighDataValue = 5
			recoveryValue = 15
			flyingAltitudeThreshold = 6000
			spaceAltitudeThreshold = 260000
		}
		ScaledVersion
		{
			type = Atmospheric
			fadeStart = 0
			fadeEnd = 0

			Material
			{
				texture = TDS/PhoenixSystem/Abraxa/Textures/Abraxa_color
				normals = TDS/PhoenixSystem/Abraxa/Textures/Abraxa_normal
				shininess = 0.01
				specular = 0.12,0.12,0.12,1

				rimPower = 2.06
				rimBlend = 0.7

				Gradient
				{
					0.0 = 1,1,1,1
					0.5 = 1,1,1,0.5
					1.0 = 1,1,1,0
               			}
			}
		}
		Atmosphere
		{
			ambientColor = 0.25,0.25,0.25,1
			enabled = true
			oxygen = false
			altitude = 54000.0
			pressureCurveIsNormalized = false
			staticPressureASL = 100.000
			temperatureSeaLevel = 217
			PressureCurve
			{
				key = 0 100.000 0 -0.0007
				key = 12000 30.817 -0.000223 -0.000223
				key = 20000 2.776 -4.22E-05 -4.22E-05
				key = 35000 0.206 -2.787075E-06 -2.787075E-06
				key = 50000 0.00917 -1.66713E-07 -1.66713E-07
				key = 58000 0 0 0
			}
			temperatureCurve
			{
				key = 0 217 0 -0.0004261126
				key = 1000 191.0 -0.000573325 -0.000573325
				key = 25000 162 -0.001877083 -0.001877083
				key = 30000 177 0 0
				key = 45000 166 0 0
				key = 50000 170 0.003746914 0
			}
			temperatureLatitudeBiasCurve
			{
				key = 0 20 0 -0.0957164
				key = 50 0 -0.950278 -0.950278
				key = 70 -30 -1.955704 -1.955704
				key = 90 -50 -0.02418368 0
			}
			AtmosphereFromGround
			{
				waveLength = 0.8595505,5114341,0.4550562,0
			}
		}
        PQS
        {
            maxQuadLengthsPerFrame = 0.03
            minLevel = 2
            maxLevel = 12
            minDetailDistance = 8
            // Custom PQS material
            materialType = AtmosphericOptimized
            Material
            {
                saturation = 1
                contrast = 3
                tintColor = 1.000,1.000,1.000,0.000
                powerNear = 0.6
                powerFar = 0.2
                groundTexStart = 0
                groundTexEnd = 1000
                steepPower = 2
                steepTexStart = 0
                steepTexEnd = 50000
                steepTex = BUILTIN/terrain_rock00
                steepBumpMap = BUILTIN/RockyGround2
                steepNearTiling = 8000
                steepTiling = 500
                lowTex = BUILTIN/SandyGround
                lowBumpMap = BUILTIN/quiet
                lowNearTiling = 2000
                lowMultiFactor = 10
                lowBumpNearTiling = 2000
                lowBumpFarTiling = 10
                midTex = BUILTIN/terrain_sand00
                midBumpMap = BUILTIN/SandyGround
                midNearTiling = 4000
                midMultiFactor = 10
                midBumpNearTiling = 8000
                midBumpFarTiling = 10
                highTex = BUILTIN/RockyGround
                highBumpMap = BUILTIN/quiet
                highNearTiling = 4000
                highMultiFactor = 10
                highBumpNearTiling = 4000
                highBumpFarTiling = 10
                lowStart = 0
                lowEnd = 0.3
                highStart = 0.8
                highEnd = 1
                globalDensity = -5E-06
                fogColorRamp = BUILTIN/desert_atmogradient
            }
            
			Mods
			{
				VertexColorMap
				{
					map = TDS/PhoenixSystem/Abraxa/Textures/Abraxa_color.png
					order = 1000
					enabled = true
				}

				VertexHeightMap
				{
					map = TDS/PhoenixSystem/Abraxa/PluginData/Abraxa_height.png
					offset = -1700
					deformity = 8000.0
					scaleDeformityByRadius = false
					order = 20
					enabled = true
				}
				VertexSimplexHeight
				{
					seed = 64198163
					deformity = 1700.0
					octaves = 12.0
					persistence = 0.5
					frequency = 4.0
					enabled = true
					order = 21
				}
				VertexSimplexHeightAbsolute
				{
					seed = 894651
					deformity = 600
					octaves = 6.0
					persistence = 0.8
					frequency = 12.0
					enabled = true
					order = 12
				}
				VertexHeightNoiseVertHeightCurve2
				{
					deformity = 980
					ridgedAddSeed = 1212
					ridgedAddFrequency = 12
					ridgedAddLacunarity = 2
					ridgedAddOctaves = 4
					ridgedSubSeed = 23234423
					ridgedSubFrequency = 12
					ridgedSubLacunarity = 2
					ridgedSubOctaves = 4
					ridgedMode = Low
					simplexHeightStart = 0
					simplexHeightEnd = 6500
					simplexSeed = 654786
					simplexOctaves = 4
					simplexPersistence = 0.600000023841858
					simplexFrequency = 12
					enabled = true
					order = 13
				}
			}
		}
		Ocean
		{
			minLevel = 1
			maxLevel = 6
			minDetailDistance = 8
			maxQuadLengthsPerFrame = 0.03
			ocean = True
			oceanColor = 0,0,0,0
			oceanHeight = 0
			density = 0.00073
			Material
			{
				color = 0,0,0,0
				colorFromSpace = 0,0,0,0
				specColor = 0,0,0,0
				shininess = 1
				gloss = 0.2
				tiling = 1000
				waterTex = BUILTIN/sea-water8
				waterTexScale = 1,1
				waterTexOffset = 0,0
				waterTex1 = BUILTIN/sea-water1
				waterTex1Scale = 1,1
				waterTex1Offset = 0,0
				bTiling = 800
				bumpMap = BUILTIN/quiet
				bumpMapScale = 1,1
				bumpMapOffset = 0,0
				displacement = 0.05
				texDisplacement = 0.31
				dispFreq = 0.15
				mix = 0.4032745
				oceanOpacity = 0.3
				falloffPower = 3
				falloffExp = 0.05
				fogColor = 0.321443439,0.611232221,0.947761178,1
				heightFallOff = 0.2
				globalDensity = -8E-06
				atmosphereDepth = 50000
				fogColorRamp = BUILTIN/blue_atmogradient
				fogColorRampScale = 1,1
				fogColorRampOffset = 0,0
				fadeStart = 20000
				fadeEnd = 60000
				planetOpacity = 1
				normalXYFudge = 1.4
				normalZFudge = 1.18
			}
			FallbackMaterial
			{
				color = 0,0,0,0
				colorFromSpace = 0,0,0,0
				specColor = 0,0,0,0
				shininess = 1
				gloss = 0.3336538
				tiling = 1000
				waterTex = BUILTIN/sea-water1
				waterTexScale = 1,1
				waterTexOffset = 0,0
				waterTex1 = BUILTIN/sea-water2
				waterTex1Scale = 1,1
				waterTex1Offset = 0,0
				fadeStart = 20000
				fadeEnd = 60000
				planetOpacity = 1
			}
			Fog
			{
				afgAltMult = 0.05
				afgBase = 0.6
				afgLerp = False
				afgMin = 0.05
				fogColorEnd = 0,0.0850000009,0.122500002,1
				fogColorStart = 0,0.340000004,0.49000001,1
				fogDensityAltScalar = -0.0008
				fogDensityEnd = 0.025
				fogDensityExponent = 1
				fogDensityPQSMult = 0.02
				fogDensityStart = 0.005
				skyColorMult = 1.1
				skyColorOpacityAltMult = 15
				skyColorOpacityBase = 0.25
				sunAltMult = 0.01
				sunBase = 0.5
				sunMin = 0.05
				useFog = True
			}
			Mods
			{
				AerialPerspectiveMaterial
				{
					atmosphereDepth = 5000
					DEBUG_SetEveryFrame = False
					globalDensity = -7.5E-06
					heightFalloff = 0.2
					oceanDepth = 0
					order = 100
					enabled = True
					name = _Material_AerialPerspective
					index = -1
				}
				OceanFX
				{
					angle = 0
					blendA = 0
					blendB = 0
					framesPerSecond = 10
					oceanOpacity = 0
					spaceAltitude = 0
					spaceSurfaceBlend = 0
					specColor = 0,0,0,0
					texBlend = 0
					txIndex = 0
					order = 100
					enabled = True
					name = OceanFX
					index = -1
					Watermain
					{
						waterTex-0 = BUILTIN/sea-water1
						waterTex-1 = BUILTIN/sea-water2
						waterTex-2 = BUILTIN/sea-water3
						waterTex-3 = BUILTIN/sea-water4
						waterTex-4 = BUILTIN/sea-water5
						waterTex-5 = BUILTIN/sea-water6
						waterTex-6 = BUILTIN/sea-water7
						waterTex-7 = BUILTIN/sea-water8
						waterTex-0 = BUILTIN/sea-water1
					}
				}
			}
		}
	}
}

**Comment by StollD** [comment]
Show me your logfiles. Otherwise this is like guessing your hard drive space based on your avatar image

**Comment by electricpantsGITHUB** [comment]
That's where I get confused.
How do I get the logfiles?

**Comment by StollD** [comment]
http://forum.kerbalspaceprogram.com/index.php?/topic/140580-122-kopernicus-release-3-february-2/&page=27#comment-2793409

**Comment by electricpantsGITHUB** [comment]
https://drive.google.com/open?id=0B5KO_W-swenrRGRkOG1fcXRmX3M

They should be there.

**Comment by StollD** [comment]
`rotationPeriod = `

KSP **hates** empty parameters. Go and yell at the creator :P

**Comment by electricpantsGITHUB** [comment]
Oh, I was leaving it blank hoping that, since it's tidally locked, it would generate it.

Guess I was wrong, lol.

**Comment by electricpantsGITHUB** [comment]
The planet works now.
You can close the issue. ;)
